### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by aknysh**
## what
* `github-commenter` initial implementation
* Command line utility for creating GitHub comments on Commits, Pull Request Reviews or Issues

## why
* Useful for CI/CD environments like `TravisCI`, `CircleCI` or `CodeFresh` to be used in build steps
* Useful for creating GitHub comments from the formatted outputs of other programs, _e.g._

```sh
terraform plan 2>&1 | github-commenter -format "Output from `terraform plan`<br>```{{.}}```"
```

### CODE REVIEW

File: examples/run_docker_with_env_vars.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+docker run -i --rm \
+        -e GITHUB_TOKEN=XXXXXXXXXXXXXXXX \
+        -e GITHUB_OWNER=cloudposse \
+        -e GITHUB_REPO=github-commenter \
+        -e GITHUB_COMMENT_TYPE=pr \
+        -e GITHUB_PR_ISSUE_NUMBER=1 \
+        -e GITHUB_COMMENT_FORMAT="My comment:<br>{{.}}" \
```
Comment (osterman) [inline examples/run_docker_with_env_vars.sh]: `<br/>`?

File: examples/run_docker_with_local_env_vars.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export GITHUB_TOKEN=XXXXXXXXXXXXXXXX
+export GITHUB_OWNER=cloudposse
+export GITHUB_REPO=github-commenter
+export GITHUB_COMMENT_TYPE=pr
+export GITHUB_PR_ISSUE_NUMBER=1
+export GITHUB_COMMENT_FORMAT="My comment:<br>{{.}}"
```
Comment (osterman) [inline examples/run_docker_with_local_env_vars.sh]: `<br/>`?

**Comment by infrabot-hippo** [comment]
[Link to Drone CI]()

**Comment by infrabot-hippo** [comment]
[Link to Drone CI]()

**Comment by infrabot-hippo** [comment]
[Link to Drone CI](https://drone.infra.hippo-private.com/Hippo-Analytics-Inc/hippo-ansible/468)

**Comment by infrabot-hippo** [comment]
[Link to Drone CI](https://drone.infra.hippo-private.com/Hippo-Analytics-Inc/hippo-ansible/470)

**Comment by infrabot-hippo** [comment]
[Link to Drone CI](https://drone.infra.hippo-private.com/Hippo-Analytics-Inc/hippo-ansible/472)

**Comment by infrabot-hippo** [comment]
POSTED BY ACCIDENT

**Comment by infrabot-hippo** [comment]
POSTED BY ACCIDENT

**Pull request merged**